app/controllers/internal_api/v1/team_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class InternalApi::V1::TeamController < InternalApi::V1::ApplicationController
```

Comment:
```suggestion
class InternalApi::V1::TeamsController < InternalApi::V1::ApplicationController
```
Can we use a plural name